index.html

```diff
@@ -39,7 +43,215 @@ <h1>I'm Alishah. Glad to see you!</h1>
                 <a href="#"><img src="./Image/Vector (1).png" alt=""></a>
                 <a href="#"><img src="./Image/Vector (4).png" alt=""></a>
             </div>
-         </section> 
-     </div>
+        </section>
+        <section class="wraper">
+            <div class="works-container fist-works-container">
+                <div class="snapshoot">
+                    <img src="./Image/SnapshootPortfolio1.png" alt="">
+                </div>
+                <div class="leftBlock">
+                    <div class="primarytext1">
+                        <h2>Tonic</h2>
+                        <div class="fram">
+                            <span class="canopy">CANOPY</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">Back End Dev</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">2015</span>
+                        </div>
+                    </div>
+                    <div class="primarytext2">
+                        <p>A daily selection of privately <br>
+                            personalized reads;no accounts or<br>
+                            sign-ups required.</p>
+                    </div>
+                    <ul class="primarytext3">
+                        <li class="lang">html</li>
+                        <li class="lang">css</li>
+                        <li class="lang JavaScript">javascript</li>
+                    </ul>
+                    <div class="primarytext4">
+                        <button class="project">See project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="works-container">
+                <div class="snapshoot">
+                    <img src="./Image/SnapshootPortfolio(3).png" alt="">
+                </div>
+                <div class="leftBlock">
+                    <div class="primarytext1">
+                        <h3 class="multi-post">
+                            Multi-Post
+                            <br>
+                            Stories
+                        </h3>
```

Comment:
- I'd like to apologize if you're confused with the instructions of the previous reviewer:
> * [ ]  I will recommend you ensure that the title of your project cards are wrapped in heading tags e.g (`<h3>`) and not a div tag because it represents the title of a project and as such the heading semantic tag should be used so that screen readers and browsers will be able to understand what every part of your website represents 👍🏽

- But the way he mentioned you use `h3` was as an example you should always try to stay consistent with the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=pSU459atAyqpbx9o-0) and use similar headings for the title of your project.
    - Kindly change the `h3` tag to `h2` just like you did with the "Tonic" heading this will ensure that all project titles look the same.